### PR TITLE
Issue #376 Shorten and simplify documentation about dev install on Wi…

### PR DESCRIPTION
**Pull request by JohanKJSchreurs**
# Issue #376 Shorten and simplify documentation about dev install on Windows

Link to issue: https://github.com/Open-EO/openeo-python-client/issues/376

### CODE REVIEW

**Comment by soxofaan** [comment]
looks good that the windows dependency hell stuff can be dropped, but what remains has little added value in the context of a "development" install (`pip install -e ...`)

**Comment by soxofaan** [comment]
I had a quick look at `conda install`, and there seems to be a `--only-deps` option, so that might be a handy solution to do `conda install` for deps only and `pip install -e` for openeo itself

**Comment by JohanKJSchreurs** [comment]
> I had a quick look at `conda install`, and there seems to be a `--only-deps` option, so that might be a handy solution to do `conda install` for deps only and `pip install -e` for openeo itself

Indeed, what was described would not have worked for development. 

I updated the instructions and tested it out.  The `--only-deps` option was indeed what we needed. After that step, installing the code in editable mode works correctly.

File: docs/development.rst [hunk 1]
```diff
@@ -270,128 +270,39 @@ Normally you can install the client the same way on Windows as on Linux, like so
...
+There is a also shorter version of the above, if you prefer.
+You can create the environment and install openeo's dependencies in one go:
 
-.. code-block::
+.. code-block:: console
 
-    # In your activate virtualenv
-    # install the wheels:
-    pip install <path to GDAL whl file> <path to fiona whl file>
+    conda create -n openeopyclient --only-deps -c conda-forge openeo
```
Comment (soxofaan) [inline docs/development.rst:304]: I would not provide this short version in the docs, doing `conda create` with `--only-deps` might get ugly if users also add other packages to the `conda create`  line

Comment (JohanKJSchreurs) [reply docs/development.rst]: Good point. I removed that paragraph.

**Pull request merged**